test/contract-tests/tests/test/amazon/awssdk/awssdk_test.py

```diff
@@ -558,12 +794,16 @@ def test_bedrock_agent_get_data_source(self):
             remote_operation="GetDataSource",
             remote_resource_type="AWS::Bedrock::DataSource",
             remote_resource_identifier="test-data-source",
+            cloudformation_primary_identifier="test-knowledge-base|test-data-source",
             request_response_specific_attributes={
                 _AWS_BEDROCK_DATA_SOURCE_ID: "test-data-source",
+                _AWS_BEDROCK_KNOWLEDGE_BASE_ID: "test-knowledge-base"
             },
             span_name="Bedrock Agent.GetDataSource",
         )
 
+    # TODO: add contract test for Lambda event source mapping resource
```

Comment:
Leaving a TODO item here for the event source mapping contract test case. When creating an event source mapping, LocalStack generates a random UUID each time which is then needed to call GetEventSourceMapping. Still looking for a solution for passing this UUID into the API call after it is generated.